unattended_installer/passwords_tool/passwordsMain.sh

```diff
@@ -86,16 +84,10 @@ function main() {
                 ;;
             "-A"|"--api")
                 api=1
-                if [ -z ${2} ]; then
-                    echo "Argument -A|--api needs a second argument"
-                    getHelp
-                    exit 1
-                fi
                 currentPassword=${2}
-                shift 2
+                shift 1
```

Comment:
Does it needs `Shift 2`? or include the $2 check?

Reply:
Done [8455a7f](https://github.com/wazuh/wazuh-packages/pull/1663/commits/8455a7f193cc2dccb41818cd5e4d406fa7018f27)